src/simtools/simulator.py

```diff
@@ -644,26 +674,36 @@ def pack_for_register(self, directory_for_grid_upload=None):
         )
         directory_for_grid_upload.mkdir(parents=True, exist_ok=True)
 
-        tar_file_name = directory_for_grid_upload.joinpath(tar_file_name)
+        # If there are more than one model version,
+        # duplicate the corsika log file to have one for each model version with the "right name".
```

Comment:
Not sure about the log file duplication. This is the best solution? Or can we have the corsika log file separate to all sim_telarray logfiles?

Reply:
Whether it is the best solution, I cannot really tell you. However, I can definitely say it is the most convenient option. The log files are fairly small and if we can always provide the full set of log files with each data file, I think it's a good idea. Otherwise the user will have to search for the corresponding CORSIKA log file separately, which is much less fun with grid tools.

That being said, we never really saved the CORSIKA log files thus far. So my argument above isn't the strongest.